audio/audio.c

```diff
@@ -223,7 +235,7 @@ static int audio_close(FAR struct file *filep)
       DEBUGASSERT(lower->ops->shutdown != NULL);
       audinfo("calling shutdown\n");
 
-      lower->ops->shutdown(lower);
+      lower->ops->shutdown(lower, upper->crefs);
```

Comment:
let's call shutdown when the last reference close and drop opencnt from shutdown prototype

Reply:
shutdown is called not only close method, but also SHUTDOWN ioctl, so I think it should be kept.